tests/lean/Reformat.lean.expected.out

```diff
@@ -32,27 +32,25 @@ def idDelta {α : Sort u} (a : α) : α :=
 /- `idRhs` is an auxiliary declaration used to implement "smart unfolding". It is used as a marker. -/
 @[macroInline, reducible]
 def idRhs (α : Sort u) (a : α) : α :=
-  a 
+  a
 
-abbrev Function.comp {α : Sort u} {β : Sort v} {δ : Sort w} (f : β → δ) (g : α → β) : α → δ :=
-  fun x => f (g x)
+abbrev Function.comp {α : Sort u} {β : Sort v} {δ : Sort w} (f : β → δ) (g : α → β) : α → δ := fun x => f (g x)
```

Comment:
Ah, that `fun` but not other terms can now be used in a single line is an interesting consequence... I think we should really introduce `ppLineUnlessUngrouped` and use it instead of `ppHardline...` but that doesn't have to be in this PR